docs/asdf-astropy/details.rst

```diff
@@ -0,0 +1,33 @@
+.. _details:
+
+=======
+Details
+=======
+
+The `asdf-astropy` package defines classes, referred to as **converters**, that
+implement the logic for serialization and deserialization of ``astropy`` types.
+Users should never need to refer to converter implementations directly. Their
+presence should be entirely transparent when processing ASDF files.
+
+ASDF makes use of abstract data type definitions called `schemas`, which typically
```

Comment:
IMO the data type definition is really the tag, which typically has an associated schema.  It's possible to have a tag without a schema but not the other way around.